worker/app/utils/settings.py

```diff
@@ -27,6 +27,9 @@ class Settings:
 
     docker_socket: str = '/var/run/docker.sock'
 
+    sockets_dir_host: str = os.getenv('SOCKETS_DIR', '/tmp/sockets')
+    sockets_dir_container: str = '/tmp/sockets'
```

Comment:
Hmm, would it be a good idea to just utilize the `host_working_dir` env? I personally think it is a good idea to keep all files zimfarm needs and generated in one place, instead of scattered around the file system.

Reply:
OK, I think variables are less clear this way but you're right, the less volumes the better.